src/components/NotesPage/NotesPageContainer.js

```diff
@@ -0,0 +1,23 @@
+import { connect } from 'react-redux';
+import { action } from 'sagas';
+import { NotesPageComponent } from 'components/NotesPage/NotesPageComponent';
+import { CHANGE_HELP_VIEW_ACTION } from 'components/NotesPage/NotesPageActions';
+
+const mapStateToProps = ({ notesReducer: { selectedNote, noteText, isHelpViewed } }) => {
```

Comment:
Why are we calling properties in state like `notesReducer` instead of just `notes` ?

Reply:
renamed